src/main/kotlin/dev/arbjerg/ukulele/utils/TextUtils.kt

```diff
@@ -0,0 +1,18 @@
+package dev.arbjerg.ukulele.utils
+
+import java.util.concurrent.TimeUnit
+
+class TextUtils {
+    companion object {
```

Comment:
Remove `companion object` and replace `class` with `object`.

https://kotlinlang.org/docs/tutorials/kotlin-for-py/objects-and-companion-objects.html